libraries/AP_Scripting/examples/ahrs-print-variances.lua

```diff
@@ -0,0 +1,13 @@
+-- This script displays the ahrs variances at 1hz
+
+function update() -- this is the loop which periodically runs)
+  vel_variance, pos_variance, height_variance, mag_variance, airspeed_variance, offset = ahrs:get_variances()
```

Comment:
let's remove the offset

Reply:
txs for the review.  I've removed the offset in the follow-up commits.